config/variables.tf

```diff
@@ -0,0 +1,42 @@
+variable "base_url" {
+  type        = string
+  description = "The base url for the keycloak server to target"
+  sensitive   = true
+}
+
+variable "client_id" {
+  type        = string
+  description = "The id of the client to be created"
+  sensitive   = true
+}
+
+variable "client_name" {
+  type        = string
+  description = "The name of the client to be created"
+  sensitive   = true
+}
+
+variable "tf_client_id" {
```

Comment:
Also I don't think these can be used for setting properties on a provider so they would need removing. Has this been run against a local keycloak instance

Reply:
They seem to be working for me, I've ran this locally and there was no issue

Reply:
Responded to previous comment about provider configuration. No issue with this using input variables